include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -3604,6 +3604,7 @@ struct OrtApi {
    *   "qnn_saver_path": File path to the QNN Saver backend library. If specified, QNN Saver will be enabled and will
    *   dump QNN API calls to disk for replay/debugging. QNN Saver produces incorrect model inference results and
    *   may alter model/EP partitioning. Use only for debugging.
+   *   "qnn_context_priority": QNN context priority, options: "low", "normal", "normal_high", "high".
```

Comment:
Is the default value "undefined"? If so, we should mention it and allow the user to explicitly set the default value without printing a warning log when they do so.

Reply:
default to normal. undefined is not allowed will cause QNN API failure.

---
In reply to: [1386887220](https://github.com/microsoft/onnxruntime/pull/18315#discussion_r1386887220) [](ancestors = 1386887220)